calculator/parser.py

```diff
@@ -0,0 +1,138 @@
+BINARY_OPERATORS = ["+", "-", "*", "/"]
+
+
+def binary_operator(lhs, rhs, op):
+    """
+    Evaluates and acts on different binary operators
+
+    Parameters
+    ----------
+    lhs : integer or float
+        The first number to be operated on, will be on the left hand side of the op
+    
+    rhs : integer or float
+        The second number to be operated on, will be on the right hand side of the op
+    
+    op : string
+        The operator that is going to be applied to the numbers
+
+    Returns
+    -------
+    integer : The final answer after after the nums are operated on
+    """
+
+    if op == "+":
+        return lhs + rhs
+
+    elif op == "-":
+        return lhs - rhs
+
+    elif op == "*":
+        return lhs * rhs
+
+    elif op == "/":
+        return lhs / rhs
+    else:
+        raise ValueError(
+            "The operator you used: {} is not supported, please try another one!".format(
+                op
+            )
+        )
+
+
+def push(e, stack):
+    """
+    Pushes the element (e) to the given stack, which mutates the stack
+
+    Parameters
+    ----------
+    e : element
+        The element to be pushed on top of the stack
+
+    stack : list
+        The stack that e is going to be pushed onto
+
+    """
+    return stack.append(e)
```

Comment:
Please do not return `None` here.  If you don't intend on returning, just remove the return statement.  return `None` is different than not returning.